docs/API/API-socket-options.md

```diff
@@ -1068,20 +1070,22 @@ undefined behavior:
 
   - **Bidirectional in Caller-Listener arrangement**: it is recommended to use
   a rule whereby you will be setting the `PBKEYLEN` exclusively either on the
-  Listener or on the Caller. The value set on the Listener will win, if set on
-  both parties.
+  Listener (the service defines the encryption rules strictly) or on the Caller
+  (the service allows all clients to freely decide about encryption, or the
+  server uses more elaborate rules for encryption).
 
-  - **Bidirectional in Rendezvous arrangement**: you have to know the passphrases
-  for both parties, as well as `PBKEYLEN`. Set `PBKEYLEN` to the same value on
-  both parties (or leave the default value on both parties, which will
-  result in 16)
+  - **Bidirectional in Rendezvous mode**: In Rendezvous mode it is not possible for a party to be "unknown", 
```

Comment:
> In Rendezvous mode it is not possible for a party to be "unknown"

What is meant here?

Reply:
When you establish a connection between two network endpoints, in case of caller-listener, it might be that the person who performs the connectio (or also an automated actor) is in control of only the caller side. In case of Rendezvous, the person who establishes the connection must be in control of both sides at once. This is because a "dedicated address" in case of caller-listener is only used by the caller (listener accepts connections from wherever they come), while in case of Rendezvous both sides use dedicated addresses (of the peer of one another).

This means that while a listener could have set PBKEYLEN to some value forcefurlly and might be unaware what the caller would do, the owner of the rendezvous connection knows this setting on both sides and knows what is happening on both sides - so none of the connection parties could be anyhow "unaware" of what the other party has set.

Reply:
Maybe better said would be something like: In Rendezvous mode it is assumed that someone who establishes a connection between two parties is in control of both of them,...

Reply:
```suggestion
  - **Bidirectional in Rendezvous mode**: In Rendezvous mode it is assumed that the settings (including `PBKEYLEN`) are known at both ends. 
```